app/src/ui/pull-request-quick-view.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import { PullRequest } from '../models/pull-request'
+import { PullRequestBadge } from './branches'
+import { Dispatcher } from './dispatcher'
+import { Button } from './lib/button'
+import { SandboxedMarkdown } from './lib/sandboxed-markdown'
+import { Octicon } from './octicons'
+import * as OcticonSymbol from './octicons/octicons.generated'
+
+interface IPullRequestQuickViewProps {
+  readonly dispatcher: Dispatcher
+  readonly pullRequest: PullRequest
+
+  /** When mouse leaves the PR quick view */
+  readonly onMouseLeave: () => void
+}
+
+export class PullRequestQuickView extends React.Component<
+  IPullRequestQuickViewProps,
+  {}
+> {
+  private baseHref = 'https://github.com/'
+
+  private renderHeader = (): JSX.Element => {
+    return (
+      <header className="header">
+        <Octicon symbol={OcticonSymbol.listUnordered} />
+        <div className="action-needed">Review requested</div>
+        <Button className="button-with-icon">
+          View on GitHub
+          <Octicon symbol={OcticonSymbol.linkExternal} />
+        </Button>
+      </header>
+    )
+  }
+
+  private renderPR = (): JSX.Element => {
```

Comment:
Do we need to specify a return type here or can we rely on inference?
```suggestion
  private renderPR = () => {
```